scripts/docker/build-image.sh

```diff
@@ -8,4 +8,9 @@ LOCAL_DISCARDABLE_IMAGE=snyk/kubernetes-monitor:local
 # should happen on merging to `staging`
 NAME_AND_TAG=${1:-$LOCAL_DISCARDABLE_IMAGE}
 
-docker build -t ${NAME_AND_TAG} .
+mkdir -vp ~/.docker/cli-plugins/
+curl --silent -L --output ~/.docker/cli-plugins/docker-buildx https://github.com/docker/buildx/releases/download/v0.6.3/buildx-v0.6.3.linux-amd64
+chmod a+x ~/.docker/cli-plugins/docker-buildx
+docker run -it --rm --privileged tonistiigi/binfmt --install all
```

Comment:
Hey @odidev what does this line do?

Reply:
Hi @ivanstanev,
Since Circle-CI runner supports only amd64 platform, these lines add [docker buildx](https://docs.docker.com/buildx/working-with-buildx/) which is required to build and release docker images for multiple architectures.

Reply:
I see but what does `tonistiigi/binfmt` do and why does it need to run privileged?

Reply:
Hi @ivanstanev,
The command enables docker buildx builder to support multiple architecture builds. You can refer official Circle-CI blog to build docker images for multiple architectures [here](https://circleci.com/blog/building-docker-images-for-multiple-os-architectures/). I have added same steps.